src/components/SelectionCard.js

```diff
@@ -7,18 +7,24 @@ const Body = styled.div`
   border-width: 1px;
 `;
 
-const SelectionCard = ({ unit }) => {
-  const { name, cost } = unit; //todo temporary will need to expand deez props more when the comp grows into something usable
+const SelectionCard = ({ unit, addFunction }) => {
+  const { name, point_cost } = unit; //todo temporary will need to expand deez props more when the comp grows into something usable
```

Comment:
```suggestion
  const { name, pointCost } = unit; //todo temporary will need to expand deez props more when the comp grows into something usable
```
TREASON (couldn't find a subtitled stormtrooper gif)

Reply:
it's `point_cost` in the data 😬